front-end/index.html

```diff
@@ -21,6 +21,7 @@
 
         <script type="text/javascript">
             window.CIMonitorVersion = '---version---';
+            document.title = document.title + ' | '+ location.host;
```

Comment:
ES6 can concatinate strins like:
```js
document.title = `CIMonitor | ${location.host}`;
```

Reply:
This is now placed outside of the `dashboard.js`, I would rather see it inside of `front-end/dashboard.js::created()`. That way it's more part of the application.

Reply:
moved it